packages/vbdec.vm/vbdec.vm.nuspec

```diff
@@ -2,9 +2,9 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>vbdec.vm</id>
-    <version>12.7.22</version>
+    <version>2.22.23</version>
```

Comment:
The previous version is 12.7.22 (higher). When updating a package you should always increase the version, as when you install the package the highest available version is installed. That is why the tests are still testing the version before the fix.

I see that the version of this tool is `VBDec v1.0.917 - compiled: 2.22.23`:
![Screenshot 2023-08-31 at 13 31 23](https://github.com/mandiant/VM-Packages/assets/16052290/b19b354e-69b8-4dd8-853a-e05288f90892)

I suspect that the version we had before is also not the tool version, but the compilation date. We should use the tool version:

```suggestion
    <version>1.0.917</version>
```

But this version would be also lower than the current one. As the current package is broken, I propose we delete it manually from myget. @mr-tz is that fine for you?

Reply:
Ah, great catch! Yes, that's fine by me.

Reply:
@tylerwhardy can you update the version to 1.0.917?

Reply:
@Ana06 Updated